### PR TITLE
feat: bifrost xcm

### PR DESCRIPTION
**Pull request by Victor-Salomon**
Quick new routes added for Bifrost parachain. 
Also, it improves Subscan error readability.

Bifrost parachain to be updated with USDC, USDT, USDC.e, USDT.e

### CODE REVIEW

File: app/src/utils/subscan.ts [hunk 1]
```diff
@@ -41,6 +41,7 @@ export const trackXcm = async (
         console.log(
           `XCM transfer not found for ${crossChainMessageHash ? `message hash: ${crossChainMessageHash}` : `extrinsic id: ${sourceChainExtrinsicIndex}`}.`,
         )
+        query.json?.message && console.log(`Additional information: ${query.json.message}`)
```
Comment (noahjoeris) [inline app/src/utils/subscan.ts:44]: Do we wanna keep the logs?

File: app/src/utils/subscan.ts [hunk 2]
```diff
@@ -59,6 +60,7 @@ export const trackXcm = async (
         const uniqueIdData: SubscanTransferResponse = uniqueIdQuery.json?.data
         if (!uniqueIdData) {
           console.log(`XCM transfer not found for unique_id: ${unique_id}`)
+          query.json?.message && console.log(`Additional information: ${query.json.message}`)
```
Comment (noahjoeris) [inline app/src/utils/subscan.ts:63]: here as well

**Review by noahjoeris — APPROVED** [review]
👍

File: app/src/utils/subscan.ts (hunk 1 above)
Comment (Victor-Salomon) [reply app/src/utils/subscan.ts:44]: Yes, we now need them. 
This comment is only displayed to improve error understanding when the query is successful but the request can't be proceeded like yesterday with the requested payment.

**Review by NunoAlexandre — APPROVED** [review]
👌🏻👌🏻👌🏻

**Pull request merged**